pycbc/filter/qtransform.py

```diff
@@ -32,12 +32,156 @@
 
 """
 
+
 from numpy import pi, ceil, log, exp
 import numpy as np
 from pycbc.strain  import next_power_of_2
 from pycbc.types.timeseries import FrequencySeries, TimeSeries
+from scipy.interpolate import (interp2d, InterpolatedUnivariateSpline)
+from pycbc.fft import ifft
+from pycbc.types import zeros
 from numpy import fft as npfft
 
+from matplotlib import use
```

Comment:
It's not a great idea to import matplotlib from within the library. If you need it. Please import from the function you'll use it in.